src/firefly/test/python/demo-notebook.ipynb

```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from FireflyClient import *\n",
```

Comment:
Also they may be trying to get away from camel casing.  At one time they decided they wanted to do it like they but I remember some discussion. Might be worth asking Tim Jenness